src/views/Contacts.vue

```diff
@@ -40,7 +40,7 @@
 				<template #icon>
 					<IconAdd :size="20" />
 				</template>
-				{{ t('contacts','New contact') }}
+				{{ t('contacts','Name') }}
```

Comment:
this is the text for the left sidebar. I do not think this should change.

could you post before/after screenshots of the app, please? :)

Reply:
@ChristophWurst could you please know me how to run this app in localhost?

Reply:
@ChristophWurst In package.json there is no command for npm run start. Is there any other ways to run it on localhost?

Reply:
See https://docs.nextcloud.com/server/latest/developer_manual/getting_started/devenv.html. You need a running Nextcloud instance for the Contacts app to work.

Reply:
@ChristophWurst my question is in package.json there is no command for running this app. You have sent me the tutorial for server repo.

Reply:
It is not possible to run this app without Nextcloud server. Nextcloud server is the platform for apps.

Reply:
https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#setup